src/app.css

```
@@ -28,17 +28,20 @@
     --lineHeight-l: calc(var(--fontSize-l) * 1.8);
     --lineHeight-xl: calc(var(--fontSize-xl) * 1.8);
     --lineHeight-2xl: calc(var(--fontSize-2xl) * 1.8);
+    --m3-body-font-family: 'Inter Variable', Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)